docs/content/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference.mdx

```diff
@@ -308,3 +313,26 @@ isolated_agents:
   </ReferenceTableItem>
 
 </ReferenceTable>
+
+### isolated_agents properties
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="isolated_agents.enabled">
+    When isolated agents is enabled, they can no longer access each others'
+    resources. See [Running multiple
+    agents](/dagster-cloud/deployment/agents/running-multiple-agents#running-multiple-agents-in-different-environments)
+  </ReferenceTableItem>
+</ReferenceTable>
+
+### agent_queues properties
+
+These properties control which requests queue are routed to the agent. See [Routing requests to specific agents](/dagster-cloud/deployment/agents/running-multiple-agents#routing-requests-to-specific-agents).
```

Comment:
> These properties control which **requests queue are routed** to the agent.

I think there may be an extra word in here - this is about the requests that are routed to the agents (and their queue), correct? Right now this reads as confusing to me.

Reply:
changed to `Specify from which queue(s) the agent will obtain requests`. is that better?

Reply:
I guess that should include a pronoun in there. 

Like `These properties specify...`

Reply:
Better! One additional suggestion:

> These settings specify the queue(s) the agent will obtain requests from.

I kept getting tripped up on `from which` - IMO this is very straightforward and less wordy.

Reply:
done. 'from which' -> i blame my french wired brain.